project_tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+proofreader src/ test/ web/
```

Comment:
If it were limited to php-cs-fixer, it would be consistent with that file; but it's also running `phpmd` and `phpcpd` which need a list of folders to work on more than an iterator of files